src/store/features/resources/resourcesSlice.ts

```diff
@@ -0,0 +1,42 @@
+import { type PayloadAction, createSlice } from "@reduxjs/toolkit";
+import storage from "redux-persist/lib/storage";
+import { PURGE } from "redux-persist";
+import { clientSignOut } from "@/store/features/auth/authSlice";
+
+export interface ResourceData {
+  id: number;
+  teamMemberId: number;
+  url: string;
+  title: string;
+  createdAt: Date;
+  updatedAt: Date;
+  addedBy: { member: [object] };
+}
+interface ResourcesState {
+  //loading:
+  resources: ResourceData[];
+}
+const initialState: ResourcesState = {
+  //loading:false
+  resources: [],
+};
+
+export const resourcesSlice = createSlice({
+  name: "resources",
+  initialState,
+  reducers: {
+    fetchResources: (state, action: PayloadAction<ResourceData[]>) => {
+      state.resources = action.payload;
+    },
+  },
+  extraReducers(builder) {
+    builder.addCase(PURGE, () => {
+      void storage.removeItem("persit:root");
+    });
+    builder.addCase(clientSignOut, () => initialState);
+  },
```

Comment:
I don't think this is necessary. It's not being called anywhere related to the resources feature, nor is the resources being persisted (and it doesn't need to be)